tests/test_dockers.py

```diff
@@ -146,13 +146,21 @@ def test_flair(self):
         )
         self.framework_invalid_test("flair")
 
-    def test_sklearn(self):
+    def test_sklearn_tabular_classification(self):
         self.framework_docker_test(
             "sklearn",
             "tabular-classification",
             "julien-c/wine-quality",
         )
 
+    def test_sklearn_tabular_regression(self):
```

Comment:
Can we keep the test in a single `test_sklearn` ?

These are very high level integration test, it makes no sense to split them since every single model should pass, or something is wrong (I mostly use them to sanity check before deploying)

Reply:
Done